tests/run-pass/fs.rs

```diff
@@ -72,4 +75,41 @@ fn main() {
     assert_eq!(ErrorKind::NotFound, remove_file(&path).unwrap_err().kind());
     // Reading the metadata of a non-existing file should fail with a "not found" error.
     assert_eq!(ErrorKind::NotFound, test_metadata(bytes, &path).unwrap_err().kind());
+
+    // Creating a directory should succeed.
```

Comment:
This `main` function is getting too long... could you move this into a separate function, and have `main` just call a bunch of separate test functions?